release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -473,6 +473,127 @@ The Local Storage Operator now has the ability to:
 [id="ocp-4-6-operators"]
 === Operators
 
+[id="ocp-4-6-operator-api"]
+==== Operator API now supported
+
+The Operator API introduced in {product-title} 4.5 as a Technology Preview
+feature is now supported and enabled by default. Previously, installing
+Operators using Operator Lifecycle Manager (OLM) required cluster administrators
+to be aware of multiple APIs, including CatalogSources, Subscriptions,
+ClusterServiceVersions, and InstallPlans. This single Operator API resource is a
+first step towards a more simplified experience discovering and managing the
+lifecycle of Operators in a {product-title} cluster.
+
+While previously only available in read-only mode, cluster administrators can
+now use the web console, in addition to the CLI, with this single API to
+interact with installed Operators. In addition, relevant OLM objects are now
+automatically labeled accordingly for the new Operator API for any Operators
+where the CSV is installed using a Subscription.
+
+Cluster administrators can discover and edit previously installed Operators
+using this API either using the web console or the CLI, for example using the
+`oc get operators` command.
+
+[id="ocp-4-6-operator-rm-tp"]
+===== Removing Technology Preview Operator API before cluster upgrade
+
+If you enabled the Technology Preview feature version of Operator API in
+{product-title} 4.5, you must disable it before upgrading to {product-title}
+4.6:
+
+.Procedure
+
+. Delete the Operator CRD:
++
+[source,terminal]
+----
+$ oc delete crd operators.operators.coreos.com
+----
+
+. Remove the `OperatorLifecycleManagerV2=true` feature gate from the OLM Operator.
+
+.. Edit the Deployment for the OLM Operator:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager \
+    edit deployment olm-operator
+----
+
+.. Remove the following flags from the `args` section in the Deployment:
++
+[source,terminal]
+----
+...
+    spec:
+      containers:
+      - args:
+...
+        - --feature-gates <1>
+        - OperatorLifecycleManagerV2=true <1>
+----
+<1> Remove these flags.
+
+.. Save your changes.
+
+. Re-enable xref:../architecture/architecture-installation.adoc#unmanaged-operators_architecture-installation[Cluster Version Operator (CVO) management] of OLM:
++
+[source,terminal]
+----
```

Comment:
@adellape the current command is just to change flag to false, but the item still is kept. the better is to remove it from spec. here is my method. FYI. @njhale appreciate you help review it.
1, check if the olm-operator is the only entry of overrides
```console
# oc get clusterversion version -o=jsonpath='{.spec.overrides[*].name}'
olm-operator cluster-network-operator
```
if only olm-operator is printed, it means it is the only entry. Or else, it is not only entry, and please record its index of overrides (its sequence number -1). For this example, it is 0 (1 -1)

2, if it is the only entry, could use the following command to remove it.
```console
# oc patch clusterversion version \
       --type json -p \
       '[{
         "op": "remove",
         "path": "/spec/overrides"
        }]'
```

3, if it is not the only entry, could use the following command to remove it (Note: use the index of the step1).
```console
# oc patch clusterversion version \
       --type json -p \
       '[{
         "op": "remove",
         "path": "/spec/overrides/<index in step1>"
        }]'
```
here is the example:
```console
# oc patch clusterversion version \
       --type json -p \
       '[{
         "op": "remove",
         "path": "/spec/overrides/0"
        }]'
```

Reply:
while I do think it's better to remove the entry entirely, IMO mindlessly running a single command is better UX than conditionally executing several. With that in mind, my preference is the existing merge command unless we can think of a shell agnostic, idempotent one-liner to "remove if exists".

Reply:
I'll leave it as the single merge command. Thank you.